src/components/JumbotronVideoComponent.vue

```diff
@@ -24,18 +30,30 @@ if (props.img) {
     }
   });
 }
+const splitOrientationToUse =
+  props.splitOrientation != undefined ? props.splitOrientation : 'vertical';
+console.log(splitOrientationToUse);
 </script>
 
 <template>
   <div ref="mainElementRef" class="jumbotron d-flex align-items-center">
     <h1 class="text-md-h1 mb-4">{{ title }}</h1>
-    <video v-if="video" class="bg-video" preload="true" autoplay muted>
-      <source :src="video" type="video/mp4" />
-    </video>
+    <div :class="`split split-${splitOrientationToUse}`">
+      <video
+        v-for="(video, i) in videos"
+        :key="i"
+        preload="true"
+        autoplay
+        muted
+        loop
+      >
+        <source :src="video" type="video/mp4" />
+      </video>
+    </div>
   </div>
 </template>
 
-<style>
+<style lang="scss">
```

Comment:
The purpose of scss is to use variables to make it easier to modify values. I don't think scss is necessary here.